scripts/gpu_drivers.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash
 
+# Disable popups to restart services after `apt-get upgrade` by restarting them automatically
+sudo sed -i "s/\#\$nrconf{restart} = 'i';/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf
+# Don't install suggested or recommended packages with `apt`
+printf "APT::Install-Suggests "0";\nAPT::Install-Recommends "0";" >> /etc/apt/apt.conf.d/99Recommended
+
 sudo apt-get update && sudo apt-get upgrade -y
+# Pin kernel version to prevent driver/kernel mismatches
+sudo apt-mark hold linux-generic linux-image-generic linux-headers-generic
+
 sudo apt-get install ubuntu-drivers-common -y
 sudo ubuntu-drivers devices
-# Edit with the desired driver number (e.g. 535)
-sudo ubuntu-drivers install <driver>
+# Edit with the desired driver number based on `ubuntu-drivers devices`
+sudo ubuntu-drivers install 535
+# Pin driver version to prevent driver/kernel mismatches
+sudo apt-mark hold nvidia-driver-535
```

Comment:
Is there a way to set up a variable $NVIDIA_DRIVER_VERSION for that magic number, that we could update in one single place?